docs/BSD.md

```diff
@@ -7,7 +7,7 @@ Tested on FreeBSD 12.1 VM in AWS EC2.
 ## Dependencies
 
 ```
-$ su root -c 'pkg update -f && pkg install -y gdb gcc gmake lmdb autoconf automake libtool git python3 emacs-nox'
+$ su root -c 'pkg update -f && pkg install -y gdb gcc gmake lmdb autoconf automake libtool git python3'
```

Comment:
LOL, BOOOOOOO

Reply:
editor 🔥 merge this PR and the 🔥  is gone ;) and I don't have to edit the line when running it with `.w !sh` in `ed(1)` ;)

Reply:
while we're at it, we should remove the preceding `$ ` and `# ` from stuff like this... I am always having to delete it with `s/^$//` or `s/^#//` before running `.w !sh` :p

Reply:
@craigcomstock IMO this just makes it slightly more annoying. Whenever I tested something on BSD, I didn't have an editor, so I need to install something, and having this command ready helped :)